src/core/packages/chrome/layout/core-chrome-layout/layouts/common/global_app_styles.tsx

```diff
@@ -20,6 +20,10 @@ const hackGlobalFieldFormattersPluginStyles = (euiTheme: UseEuiTheme['euiTheme']
   }
 
   // Styles applied to the span.ffString__emptyValue from FieldFormat class that is used to visually distinguish empty string values when rendering string values as HTML in Kibana field formatters
```

Comment:
Looks like this comment applies to the `ffString__emptyValue` style below. Probably best to move it down and add a separate one that explains what the new style does.

Reply:
Tweaked in [ef2b84e](https://github.com/elastic/kibana/pull/228086/commits/ef2b84edef8420f75f17673b32531e4358bfa2bd), sorry for an oversight!